sources/packages/web/src/components/generic/COEStatusBadge.vue

```diff
@@ -0,0 +1,85 @@
+<!--
+COE Status Badge
+-->
+<template>
+  <v-badge :color="backGroundColor" :text-color="textColor" :class="badgeClass">
+    <template v-slot:badge>
+      <font-awesome-icon
+        :icon="['fas', 'circle']"
+        class="mr-1"
+        :color="iconColor"
+      />
+      <span class="text-uppercase">{{ label }}</span>
+    </template>
+  </v-badge>
+</template>
+<script lang="ts">
+import { ref, onMounted, watch } from "vue";
+import { COEStatus } from "@/types";
+import {
+  COLOR_BLACK,
+  COLOR_WHITE,
+  COLOR_BANNER_SUCCESS,
+  COLOR_BANNER_WARNING,
+  COLOR_BANNER_ERROR,
+} from "@/constants";
+
+export default {
+  props: {
+    status: {
+      type: String,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const label = ref("");
+    const badgeClass = ref("");
+    const textColor = ref(COLOR_BLACK);
+    const backGroundColor = ref(COLOR_WHITE);
+    const iconColor = ref("");
+
+    const setStyles = () => {
+      label.value = props.status;
```

Comment:
props.status can be directly accessed in template. Please remove the extra variables added.

Reply:
props.status used instead of label.